.tool-versions

```diff
@@ -0,0 +1 @@
+python 3.11.1
```

Comment:
.tool-verions can be added to .gitignore?

Reply:
nice catch, it is now ignored